From 93625a9e109f5a6873e14fd0c15ad3a4f1e4c74f Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sun, 23 Jun 2019 16:31:02 +0100
Subject: [PATCH] dgit-maint-{merge,debrebase}(7): Suggest --name-status
 --diff-filter

This is better than just --stat for finding new and deleted files in a
new upstream release, which may need accounting for in d/copyright.

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 7 ++++---
 dgit-maint-merge.7.pod     | 2 +-
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index b91ed164..6ffa2301 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -375,9 +375,10 @@ release:
 
 =back
 
-Pass I<--stat> just to see the list of changed files, which is useful
-to determine whether there are any new or deleted files that may need
-accounting for in your copyright file.
+Additionally pass I<--name-status> and I<--diff-filter=ADR> to see
+just the list of added or removed files, which is useful to determine
+whether there are any new or deleted files that may need accounting
+for in your copyright file.
 
 If you obtained a tarball from upstream, you are ready to try a build.
 If you merged a git tag from upstream, you will first need to generate
diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index c20a2525..31b0393a 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -371,7 +371,7 @@ accounting for in your copyright file:
 
 =over 4
 
-    % git diff --stat master..1.2.3 -- . ':!debian'
+    % git diff --name-status --diff-filter=ADR master..1.2.3 -- . ':!debian'
 
 =back
 
-- 
2.11.0

